mindjunkies/lecture/tests/test_models.py

```
@@ -1,81 +1,219 @@
 import pytest
-from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
 from django.utils.text import slugify
+from model_bakery import baker
+from django.db.utils import IntegrityError
+import cloudinary
```

Ruff (0.8.2)

`cloudinary` imported but unused

Remove unused import: `cloudinary`

(F401)